frontend/src/editor/TranscriptionEditor.tsx

```diff
@@ -73,8 +75,20 @@ export default function TranscriptionEditor() {
     return () => YjsEditor.disconnect(editor);
   }, [editor]);
 
+  useEffect(() => {
+    const preventCtrlS = (e: KeyboardEvent) => {
+      const ctrlOrCmd = window.navigator.platform.match("Mac") ? e.metaKey : e.ctrlKey
+      if (ctrlOrCmd && e.key === 's') {
+        e.preventDefault();
+        console.log('CommandOrControl + S prevented – we automatically save the document anyways');
+      }
+    };
+    document.addEventListener('keydown', preventCtrlS);
+    return () => document.removeEventListener('keydown', preventCtrlS)
+  })
```

Comment:
This should have a dependency key or it will get executed every time, right?

Reply:
oops, good catch :D